content/blog/eval-testing.md

```diff
@@ -0,0 +1,294 @@
++++
+title = "Eval Testing"
+slug = "eval-testing"
+date = 2025-05-06T20:29:20+05:30
+image = "/images/2025/eval-testing/cover-image.png"
+draft = false
+authors = ["Bhumika Shah", "Dhruvin Kavathiya"]
+description = "This blog chronicles a team's journey in evaluating LLM outputs, transitioning from Eval.ai to a hybrid framework combining DeepEval's synthetic test generation with LangSmith's real-time observability. The result is a scalable, TypeScript-compatible evaluation system tailored for RAG applications and production-grade AI models."
+tags = ["Eval Testing", "Deep Eval", "LangSmith"]
+categories = ["Eval Testing", "Deep Eval", "LangSmith"]
+type = ""
++++
+
+Our Journey with LLM Evaluation Frameworks: From Eval.ai to a DeepEval-LangSmith Hybrid Approach
+
+_\"What gets measured gets improved.\" - Peter Drucker_
+
+{{< figure src="/images/2025/eval-testing/llm-journey.png" >}}
+
+## **Introduction**
+
+In the rapidly evolving landscape of Large Language Models (LLMs),
+ensuring the quality and reliability of model outputs isn\'t just good
```

Comment:
do we need the `\` before `'`s?
if not, we can remove it, markdown will support `'` without escaping

Reply:
let's address this in other places on the file too